oca_custom/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* Sylvain LE GAL (https://twitter.com/legalsylvain)
+* Benoit GUILLOT (benoit.guillot@akretion.com)
+* Stéphane Bidoul (stephane.bidoul@acsone.eu)
+* Vicent Cubells (vicent.cubells@tecnativa.com)
+* Cristina Martin R.
```

Comment:
maybe add an email?

Reply:
Solved. We try to solve all on github, so that is the reason because sometimes we don't add the email on readme.

Reply:
@cristinamartinrod don't add email. @elicoidal we don't want anymore that people contact us by email for leeching, so that's why we don't add on any README. There's no obligation either.

Reply:
fine by me: was just a suggestion